themes/default/content/blog/deprecating-non-project-mode/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Deprecating non-project mode from DIY backends"
+date: 2024-02-19
+meta_desc: Read about the plans for the removal of non-project mode.
+authors:
+    - fraser-waters
+tags:
+    - features
+    - pulumi-releases
+---
+
+Last year we released [project support for self-managed backends](/blog/project-scoped-stacks-in-self-managed-backend/), we now plan to remove support for non-project mode.
```

Comment:
It feels like more text would be helpful here. Why are we now deprecating non-project mode? While the linked blog explains things, it feels like we should take a moment and help provide some context/background for those people who haven't read (or don't wish to read) the other blog post. Right now this first paragraph is too abrupt (IMO).

Reply:
Added a bit more. This is a systems simplification, we don't think there's any use cases for non-project mode and that project mode is better for everyone. Removing it like this lets us push any stragglers towards it to have a better experience, and makes the system less costly to understand (the backends are much more consistent, don't need to know about non-project mode and the oddities it entails), and less costly to maintain (we can remove a load of code that really no one needs anymore).